file_paths.md

```diff
@@ -252,6 +277,10 @@ user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        modules *                             # contains pxp-agent modules (ruby binaries)
+            pxp-module-puppet *               # executes Puppet and provides results of Puppet runs
```

Comment:
Same comment above about `pxp-module-puppet` removing the `*`.  Packaging should create the parent `modules` directory, but I don't think `pxp-module-puppet` comes down until after the puppet agent first runs and configures the agent node, right?